subsys/bluetooth/mesh/transport.c

```diff
@@ -1001,10 +1001,8 @@ static inline int32_t ack_timeout(struct seg_rx *rx)
 	/* The acknowledgment timer shall be set to a minimum of
 	 * 150 + 50 * TTL milliseconds.
```

Comment:
Is this comment correct now?

Reply:
Seems so, default values are still the same.